src/components/SettingsForm/index.tsx

```diff
@@ -27,41 +27,33 @@ const SettingsForm = () => {
 			<fieldset>
 				<legend>Pomodoro durations</legend>
 				<small>Measured in minutes</small>
-				<Styled.FormControl>
-					<label htmlFor="workDuration">Work</label>
-					<input
-						type="number"
-						min="5"
-						max="50"
-						id="workDuration"
-						value={ secondsToMinutes(durationWork) }
-						onChange={ ({ target }) => setDurationWork(minutesToSeconds(target.valueAsNumber)) }
-					/>
-				</Styled.FormControl>
 
-				<Styled.FormControl>
-					<label htmlFor="shortBreakDuration">Short Break</label>
-					<input
-						type="number"
-						min="1"
-						max="30"
-						id="shortBreakDuration"
-						value={ secondsToMinutes(durationBreakShort) }
-						onChange={ ({ target }) => setDurationBreakShort(minutesToSeconds(target.valueAsNumber)) }
-					/>
-				</Styled.FormControl>
+				<SettingsFormPomodoroDuration
+					id="workDuration"
+					title="Work"
+					value={ durationWork }
+					onChange={ setDurationWork }
+					valueMin={ 5 }
+					valueMax={ 50 }
+				/>
 
-				<Styled.FormControl>
-					<label htmlFor="longBreakDuration">Long Break</label>
-					<input
-						type="number"
-						min="3"
-						max="50"
-						id="longBreakDuration"
-						value={ secondsToMinutes(durationBreakLong) }
-						onChange={ ({ target }) => setDurationBreakLong(minutesToSeconds(target.valueAsNumber)) }
-					/>
-				</Styled.FormControl>
+				<SettingsFormPomodoroDuration
```

Comment:
I think the name of the component isn't so descriptive

Reply:
What do you suggest?

Reply:
I mean, if the component function is an input field, so the name isn't refer to it

Reply:
I named it thinking of its parent component, being the `SettingsForm`, so I guess the name refers to this component being a child of the said component.